mlflow/server/js/src/experiment-tracking/components/RunView.js

```diff
@@ -260,6 +261,16 @@ export class RunViewImpl extends Component {
 
         {/* Metadata List */}
         <Descriptions className='metadata-list'>
+          {runName !== '' && (
```

Comment:
```suggestion
          {(
```



We could always show a run ID:

<img width="1129" alt="image" src="https://user-images.githubusercontent.com/17039389/169432157-c401ecf8-feb6-4a35-a3a9-36cbcf78fc5e.png">

Reply:
@BenWilson2 @treyworley @dbczumar @WeichenXu123 Thoughts on this?

Reply:
I think having the static rendering at all times is probably less confusing and less moving parts in the JS code. +1 to always show.

Reply:
I don't have a strong preference, either. It seems cleaner to only show Run ID when a run name exists, but I am no UI/UX expert

Reply:
> the static rendering at all times is probably less confusing

Agree. I'm leaning towards always show now.